From 39857718f4df9b40a7acf38b73140fd86c628ef0 Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Mon, 30 Apr 2012 09:07:38 -0300
Subject: [PATCH] fw_env: allow it to build without a config

We need to build u-boot tools without a board configuration for the
target packages.
fw_env just uses config.h to define the default environment of the
created image, so it really isn't mandatory.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
 tools/env/fw_env.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/tools/env/fw_env.c b/tools/env/fw_env.c
index e292d2b..2bb061a 100644
--- a/tools/env/fw_env.c
+++ b/tools/env/fw_env.c
@@ -45,8 +45,6 @@
 
 #include "fw_env.h"
 
-#include <config.h>
-
 #define WHITESPACE(c) ((c == '\t') || (c == ' '))
 
 #define min(x, y) ({				\
-- 
1.7.3.4

